test/core.test.js

```diff
@@ -133,10 +145,14 @@ describe('core', () => {
 
     assert.deepEqual(agent._cache.a, {})
     assert.deepEqual(agent._cache.l, {})
+
+    stub.restore()
   })
 
   it('send success', () => {
     agent.send('pageview', location.href)
+    agent.active = true
```

Comment:
Is this correct? or is there any good solution?

`request.get()` onload callback turn active flag on.
If we did not set flag active and start listen, spy does not work.